--- a/linux/CMakeLists.txt	2025-12-27 12:00:00.000000000 +0000
+++ b/linux/CMakeLists.txt	2025-12-27 12:00:01.000000000 +0000
@@ -14,12 +14,14 @@
   FetchContent_Declare(
     sqlite3
     URL https://sqlite.org/2025/sqlite-autoconf-3510100.tar.gz
+    URL_HASH SHA256=4f2445cd70479724d32ad015ec7fd37fbb6f6130013bd4bfbc80c32beb42b7e0
     DOWNLOAD_EXTRACT_TIMESTAMP NEW
   )
 else()
   FetchContent_Declare(
     sqlite3
     URL https://sqlite.org/2025/sqlite-autoconf-3510100.tar.gz
+    URL_HASH SHA256=4f2445cd70479724d32ad015ec7fd37fbb6f6130013bd4bfbc80c32beb42b7e0
   )
 endif()
 FetchContent_MakeAvailable(sqlite3)
